hardhat.config.js

```diff
@@ -0,0 +1,34 @@
+// Hardhat configuration
+// See (https://hardhat.org/config/) for more details.
+
+// const { MNEMONIC } = require('./test.env')
+// const MNEMONIC = 'novel mobile inform nurse circle spoon cricket soup crowd clip hawk glad'
+
+module.exports = {
+  defaultNetwork: 'hardhat',
+  networks: {
+    hardhat: {
+      chainId: 31337,
+      accounts: [
+        {
+          privateKey:
+            process.env.PRIVATE_KEY ||
```

Comment:
You could require the `test-helpers/src/hardhat.ts` here, and use the imported PK